taskcluster/fenix_taskgraph/util/scriptworker.py

```diff
@@ -221,15 +225,13 @@ def generate_beetmover_artifact_map(config, job, **kwargs):
                     file_config,
                     field,
                     job["label"],
-                    locale=locale,
-                    path_platform=platform,
-                    version=config.params["version"],
+                    locale=locale
                 )
 
             # This format string should ideally be in the configuration file,
             # but this would mean keeping variable names in sync between code + config.
             destinations = [
-                "{s3_bucket_path}/{dest_path}/{locale_prefix}/{filename}".format(
```

Comment:
I was thinking about this; this is probably because we don't do single locale repacks for Android :)